radixdlt/build.gradle

```diff
@@ -139,7 +139,7 @@ configurations.all {
 }
 
 dependencies {
-    compile 'com.radixdlt:radix-engine-library:rc~1.0-beta.17-SNAPSHOT'
+    compile 'com.radixdlt:radix-engine-library:feature~epoch-high-view-SNAPSHOT'
```

Comment:
If changes for radix-engine-library will be merged first, this change is unnecessary. Also, it potentially may break other PR's.

Reply:
yes, will change this before merging